packages/sveltekit/test/client/vendor/lookUpCache.test.ts

```diff
@@ -0,0 +1,68 @@
+import * as utils from '@sentry/utils';
+import { vi } from 'vitest';
+
+import { isRequestCached } from '../../../src/client/vendor/lookUpCache';
+
+let scriptElement: {
+  textContent: string;
+  getAttribute: (name: string) => string | null;
+} | null;
+
+vi.spyOn(utils, 'getDomElement').mockImplementation(() => {
+  return scriptElement;
+});
```

Comment:
We could have also used jsdom maybe but I think this is totally fine: https://github.com/getsentry/sentry-javascript/blob/568bc7ccab750f71ac6af184f458897d88165a2c/packages/nextjs/test/performance/client.test.ts#L83

Reply:
I'll stick with this shim for the time being as I only need this so that tests don't crash.

Reply:
Yup 👍 [3840964](https://github.com/getsentry/sentry-javascript/pull/8391/commits/384096486091670f3c415db5cf85d6bae3fea024)